src/main/java/sirius/web/controller/BasicController.java

```diff
@@ -218,7 +218,7 @@ public void onApiError(WebContext webContext, HandledException error, Format for
         if (webContext.isResponseCommitted()) {
             WebServer.LOG.WARN("""
                                        Cannot send service error for: %s - %s
-                                       As a partially successful response has already been created and committed!
+                                       A partially successful response has already been created and committed!
                                        """, webContext.getRequest().uri(), error.getMessage());
```

Comment:
Stacktrace?

Reply:
ExecutionPoint.snapshot().toString() will do the trick...